Tasks/PublishSymbolsV2/PublishSymbols.ts

```diff
@@ -37,7 +38,16 @@ export async function run(clientToolFilePath: string): Promise<void> {
         let AsAccountName = tl.getVariable("ArtifactServices.Symbol.AccountName");
         let symbolServiceUri = "https://" + encodeURIComponent(AsAccountName) + ".artifacts.visualstudio.com"
         let personalAccessToken = tl.getVariable("ArtifactServices.Symbol.PAT");
-        const connectedServiceName = tl.getInput("ConnectedServiceName", false);        
+        const connectedServiceName = tl.getInput("ConnectedServiceName", false);
+        const manifest = tl.getInput("Manifest", false); 
+        if(manifest && !fileExists(manifest)) {
+            throw new Error(tl.loc("ManifestFileNotFound", manifest));
+        }
+        else
+        {
+            tl.debug("Manifest file found at: " + manifest);
```

Comment:
looks like we will have debug message even if no manifest provided

Reply:
Thank you for correction. Here is the refactor PR https://github.com/microsoft/azure-pipelines-tasks/pull/20834. Please review and will be happy to know improvements as well.